shared/src/convention/conventionRefinements.ts

```diff
@@ -76,6 +77,22 @@ export const minorBeneficiaryHasRepresentative = ({
   );
 };
 
+export const validateBeneficiaryAddressAndParse = (
+  convention: ConventionDto,
+) => {
+  const ruleAppliesFrom = new Date("2024-09-30");
```

Comment:
```suggestion
  const ruleAppliesFrom = new Date("2024-10-01");
```

sinon les conventions d'aujourd'hui poseront un problème de schéma.
Il faudra prévoir la MEP dans la journée